src/puj/push/instructions/numeric.clj

```diff
@@ -21,8 +21,18 @@
      (i/simple-instruction :int-mult [:int :int] :int 0 *)
      {:puj.push.instruction/doc  "Pushes the product of the top 2 ints onto the int stack."})
 
+   ; @TODO :int-div - what kind of protected division should we use? push-like, which would no-op if divide by zero, or analytic quotient, as described here: https://push-language.hampshire.edu/t/gptp-2019-notes/
+
+   (with-meta
+     (i/simple-instruction :int-mod [:int :int] :int 0 mod)
```

Comment:
I believe we need to use a custom function to guard against mod by zero, right?

Reply:
You are correct! In Clojush, we just no-op, which I don't like for the same reasons I don't like no-op for divide-by-zero. Maybe mod by zero should give zero?

I also remembered that Clojure's `mod` sometimes gives `##INF` and `##NAN` when modding large numbers 😭 . See this: https://push-language.hampshire.edu/t/inf-or-nan-on-float-stack/1540?u=thelmuth

That means we need to to something to ensure they don't appear on the stacks. What I did for Clojush, which might not be the best solution, is here: https://github.com/lspector/Clojush/pull/292/files

All told, maybe we need to have more discussion about what we want the desired behavior to be before adding `int_mod`. I propose, for now, that I replace this with a TODO like with `int_div`, and point to this discussion so we remember when coding it.

Reply:
I pushed a commit that does this, but not sure about how to mark this as resolved.

Reply:
👍